cot/src/project.rs

```diff
@@ -1379,7 +1480,8 @@ mod sealed {
 /// 2. [`WithConfig`]
 /// 3. [`WithApps`]
 /// 4. [`WithDatabase`]
-/// 5. [`Initialized`]
+/// 5. [`WithCache`]
```

Comment:
I'm still unsure about cache being its own stage. I can't image anything that would NEED cache to exist in the bootstrap phase. I think having it in database step makes more sense, but also, I wouldn't like `with_database` to take cache store as part of the input. Maybe we have to rethink the bootstrap process to be more flexible, thoughts @m4tx?